### PR TITLE
Reset file

### PR DESCRIPTION
**Pull request by anadis504**

### CODE REVIEW

File: frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx [hunk 1]
```diff
@@ -211,6 +241,26 @@ const MonacoEditor = ({
...
           <ServiceConnected service={currentService} />
+          <Button
+            style={{ marginLeft: 25 }}
+            className={ classes.resetButton }
+            variant="outlined"
+            size="small"
+            disabled={pullLoading || mutationSaveLoading}
+            onClick={handleReset}
+          >
+            Discrad All Changes
```
Comment (nipponpaltsu) [inline frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx:252]: - [ ] fix typo: Discrad -> Discard

File: frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx [hunk 2]
```diff
@@ -211,6 +241,26 @@ const MonacoEditor = ({
...
+          </Button>
+          <Button
+            style={{ marginLeft: 25 }}
+            className={ classes.resetButton }
+            variant="outlined"
+            size="small"
+            disabled={pullLoading || mutationSaveLoading}
+            onClick={handleResetFile}
+          >
+            Discrad File Changes
```
Comment (nipponpaltsu) [inline frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx:262]: - [ ] fix typo: Discrad -> Discard

**Review by nipponpaltsu — CHANGES_REQUESTED** [review]
Never mind, wrong account....

File: frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx (hunk 1 above)
Comment (lauripalonen) [inline frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx]: - [x] fix typo; Discrad -> Discard

File: frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx (hunk 2 above)
Comment (lauripalonen) [inline frontend/src/components/editView/MonacoEditor/MonacoEditor.tsx]: - [x] fix typo; Discrad -> Discard

**Review by lauripalonen — CHANGES_REQUESTED** [review]
Looking good! Two minor typos spotted, see comments! :)

**Pull request merged**